.gitignore

```diff
@@ -4,4 +4,6 @@ src/seedsigner.egg-info/
 .nova
 .vscode
 src/seedsigner/models/settings_definition.json
-*.mo
```

Comment:
If we do strip out the *.mo files, we should add this back in.

Reply:
I believe that I had added this so that pulling from the repo was sufficient (as in building via seedsigner-os) without adding the post-build changes to seedsigner-os so that these files would get built automatically.

Reply:
I haven't tried altering the SeedSigner OS build process, but there already is [a python processing step](https://github.com/SeedSigner/seedsigner-os/blob/main/opt/pi0/board/post-build.sh#L91) so hopefully [adding the .mo compilation](https://github.com/kdmukai/seedsigner/blob/initial_multilanguage/babel/README.md#for-packagers) (see the "For Packagers" section at the end) wouldn't be an issue.

Tagging in @newtonick for any thoughts.

Reply:
Compiled *.mo files are now in the new seedsigner-translations repo for easy pulling during build.